app/controllers/iterations_controller.rb

```diff
@@ -53,7 +53,8 @@ def get_customer_feedback
         pf.iteration = iter
         pf.save
 
-        url = "http://localhost:3000/feedback/#{eng.id}/#{iter.id}"
+        host = request.env["REQUEST_URI"].split("/")[2]
```

Comment:
why not `request.host` or `request.host_with_port` ?
http://apidock.com/rails/ActionDispatch/Http/URL/host_with_port